lib/gtfs.ex

```diff
@@ -47,10 +47,10 @@ defmodule Gtfs do
   end
 
   # Timepoint IDs on a route, sorted in order of stop sequence
-  @spec timepoint_ids_on_route(Route.id()) :: [StopTime.timepoint_id()]
-  @spec timepoint_ids_on_route(Route.id(), GenServer.server()) :: [StopTime.timepoint_id()]
-  def timepoint_ids_on_route(route_id, server \\ __MODULE__) do
-    GenServer.call(server, {:timepoint_ids_on_route, route_id})
+  @spec timepoints_on_route(Route.id()) :: [Timepoint.id()]
+  @spec timepoints_on_route(Route.id(), GenServer.server()) :: [Timepoint.id()]
```

Comment:
These return `[Timepoint.t()]` now. (I wish elixir could specify types in messages between processes so this could be caught automatically...)

Reply:
Good catch.